src/data/reducers/shopify.js

```diff
@@ -207,6 +213,30 @@ const removeLineItemInCart = (checkoutId, lineItemId) => {
   };
 };
 
+// Add discount code to checkout
+const addDiscountToCheckout = (checkoutId, discountCode) => {
+  return async dispatch => {
+    const res = await client.checkout.addDiscount(checkoutId, discountCode);
+    dispatch({
+      type: ADD_DISCOUNT,
+      payload: res,
+    });
+    return res;
+  };
+};
+
+// Remove discount code from checkout
+const removeDiscountFromCheckout = checkoutId => {
```

Comment:
Going to need to add error handling, not sure the best method to do this

Reply:
we don't :) 

i'll add later if I have the motivation to, but honestly I don't think it's worth it for a feature that probably won't be used much (and you can do it on the Shopify checkout anyways)

If you were to do it though, you'd probably have to modify what the Redux store looks like